### PR TITLE
DRY things up a bit here and there

### PR DESCRIPTION
**Pull request by maratoid**

### CODE REVIEW

File: bin/kraken-connect.sh [hunk 1]
```diff
@@ -13,55 +13,54 @@ KRAKEN_ROOT=$(dirname "${BASH_SOURCE}")/..
...
+  kraken_data:/kraken_data/${KRAKEN_CLUSTER_NAME}/ansible.inventory \
+  "${KRAKEN_ROOT}/bin/clusters/${KRAKEN_CLUSTER_NAME}/ansible.inventory"
+docker cp \
+  kraken_data:/kraken_data/${KRAKEN_CLUSTER_NAME}/terraform.tfstate \
+  "${KRAKEN_ROOT}/bin/clusters/${KRAKEN_CLUSTER_NAME}/terraform.tfstate"
+docker cp \
+  ${kraken_container_name}:/opt/kraken/terraform/${KRAKEN_CLUSTER_TYPE}/${KRAKEN_CLUSTER_NAME}/terraform.tfvars \
+  "${KRAKEN_ROOT}/bin/clusters/${KRAKEN_CLUSTER_NAME}/terraform.tfvars"
+docker cp \
+  kraken_data:/kraken_data/kube_config \
```
Comment (spiffxp) [inline bin/kraken-connect.sh]: were it not for this exception, this whole block of `docker cp`'s could be rolled up into a pretty easy for loop, remind me again why we're not dumping this file into the cluster-specific folder?

Comment (maratoid) [inline bin/kraken-connect.sh]: we are. ${KRAKEN_ROOT}/bin/clusters/${KRAKEN_CLUSTER_NAME}/id_rsa

File: bin/kraken-down.sh [hunk 2]
```diff
@@ -12,54 +12,29 @@ KRAKEN_ROOT=$(dirname "${BASH_SOURCE}")/..
...
-inf "Tearing down kraken cluster:\n  'docker run -d --name ${kraken_container_name} --volumes-from kraken_data samsung_ag/kraken \
-  /opt/kraken/terraform-down.sh --clustertype ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}'"
+inf "Tearing down kraken cluster:\n  'docker run -d --name ${kraken_container_name} \
+  --volumes-from kraken_data samsung_ag/kraken /opt/kraken/terraform-down.sh --clustertype \
+  ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}'"
 
 docker run -d --name ${kraken_container_name} --volumes-from kraken_data \
-  samsung_ag/kraken bash -c "/opt/kraken/terraform-down.sh --clustertype ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}"
+  samsung_ag/kraken bash -c "/opt/kraken/terraform-down.sh --clustertype ${KRAKEN_CLUSTER_TYPE} \
+  --clustername ${KRAKEN_CLUSTER_NAME}"
```
Comment (spiffxp) [inline bin/kraken-down.sh]: can the `inf` and `docker run` lines use a variable for the command instead of copy-paste?

File: bin/kraken-up.sh [hunk 3]
```diff
@@ -9,81 +9,35 @@ set -o pipefail
...
-  inf "Removing old kraken_cluster container:\n   'docker rm -f ${kraken_container_name}'"
+  inf "Removing old kraken_cluster container:\n   \
+    'docker rm -f ${kraken_container_name}'"
   docker rm -f ${kraken_container_name} &> /dev/null
 fi
 
-inf "Building kraken container:\n  'docker build -t samsung_ag/kraken -f \"${KRAKEN_ROOT}/bin/build/Dockerfile\" \
-  \"${KRAKEN_ROOT}\"' "
+inf "Building kraken container:\n  \
+  'docker build -t samsung_ag/kraken -f \"${KRAKEN_ROOT}/bin/build/Dockerfile\" \"${KRAKEN_ROOT}\"' "
```
Comment (spiffxp) [inline bin/kraken-up.sh]: can the `inf` and `docker build` lines use a variable for the command instead of copy-paste?

File: bin/kraken-up.sh [hunk 4]
```diff
@@ -9,81 +9,35 @@ set -o pipefail
...
 docker build -t samsung_ag/kraken -f "${KRAKEN_ROOT}/bin/build/Dockerfile" "${KRAKEN_ROOT}"
 
 # run cluster up
-inf "Building kraken cluster:\n  'docker run -d --name ${kraken_container_name} --volumes-from kraken_data samsung_ag/kraken /opt/kraken/terraform-up.sh --clustertype ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}'"
-docker run -d --name ${kraken_container_name} --volumes-from kraken_data samsung_ag/kraken bash -c "/opt/kraken/terraform-up.sh --clustertype ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}"
+inf "Building kraken cluster:\n  'docker run -d --name ${kraken_container_name} \
+  --volumes-from kraken_data samsung_ag/kraken /opt/kraken/terraform-up.sh --clustertype \
+  ${KRAKEN_CLUSTER_TYPE} --clustername ${KRAKEN_CLUSTER_NAME}'"
+docker run -d --name ${kraken_container_name} --volumes-from kraken_data samsung_ag/kraken \
```
Comment (spiffxp) [inline bin/kraken-up.sh]: can the `inf` and `docker run` lines use a variable for the command instead of copy-paste?

**Comment by spiffxp** [comment]
lgtm

**Pull request merged**